module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -252,22 +296,19 @@ public function getMergedConfig(string $key = '', bool $flatten = false): array
 
         while (!empty($currentTheme)) {
             $currentThemeSet = array_merge(
-                (array)$currentTheme,
                 $allThemeInfo[$currentTheme]['mixins'] ?? [],
+                (array)$currentTheme,
```

Comment:
Related to my comment above, why is the order of mixin vs. theme changing here? Do we expect mixins to override the theme they are mixed into, or do we expect the current theme to override its own mixins? I'm not sure what's preferable (I can see arguments for both approaches), but we should have a test case for this if we don't already, just to be sure the behavior is well-defined and doesn't get accidentally changed in future.

Reply:
I would expect mixins to override the theme they are included in. I would also expect mixins to override earlier mixins, similar to CSS. I see now that my code doesn't reflect this expectation.

Reply:
Thanks, @crhallberg, I agree! I guess if you want to override a setting from a mixin, you would have to create a local mixin to load subsequent to the mixin you are overriding.